repositories/user_repo.py

```
@@ -0,0 +1,21 @@
+import json, os
+from typing import Optional
+DB_FILE = "./DB/users.json"
+
+def get_users():
+    if not os.path.exists(DB_FILE):
+        return []
+    with open(DB_FILE, "r", encoding="utf-8") as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)